contrib/sync-sv-run.sh

```diff
@@ -66,8 +70,10 @@ if [[ ! -z "$BLOCKCHAIN_TIME" ]]; then
     fi
     # upload a current block_log
     cd $HOME
-    aws s3 cp blockchain/block_log s3://$S3_BUCKET/block_log-intransit
-    aws s3 cp s3://$S3_BUCKET/block_log-intransit s3://$S3_BUCKET/block_log-latest
+    if [[ ! "$IS_BROADCAST_NODE" ]]; then
+      aws s3 cp blockchain/block_log s3://$S3_BUCKET/block_log-intransit
+      aws s3 cp s3://$S3_BUCKET/block_log-intransit s3://$S3_BUCKET/block_log-latest
```

Comment:
would be a good idea to make sure that previous `aws s3 cp` was successful, i.e. aws s3 cp 1 && aws s3 cp 2
(to avoid copying incomplete intransit over working (previous) latest)